convert.py

```diff
@@ -438,7 +445,6 @@ def __init__(self, fname_tokenizer: Path, fname_added_tokens: Path | None) -> No
         self.vocab_size: int = self.vocab_size_base + len(self.added_tokens_list)
         self.fname_tokenizer = fname_tokenizer
         self.fname_added_tokens = fname_added_tokens
-
```

Comment:
Why remove this blank line?

Reply:
Oh this was a mistake I made while modifying the code. Thanks for catching it.